### PR TITLE
Ut coverage improvement

### PR DESCRIPTION
**Pull request by skywojciechowskim**
Summary: UTs for classes, where sink mock is not required
Type: Feature
Test Plan: UT
Jira: RIALTO-17

### CODE REVIEW

File: source/Timer.h [hunk 1]
```diff
@@ -1,285 +1,72 @@
...
-     */
-    void postTimerMessage(TimerMessage *message, bool urgent = false);
+private:
+    void doCancel();
 
-    /**
-     * @brief Timer name.
-     */
-    std::string m_name;
+private:
```
Comment (skyshaha1) [inline source/Timer.h:63]: I think you may have added this access modifier by mistake, don't think it's needed

Comment (skyshaha1) [inline source/Timer.h:63]: I think you may have added this access modifier by mistake, don't think it's needed

File: source/ITimer.h [hunk 2]
```diff
@@ -0,0 +1,89 @@
+/*
```
Comment (aczs) [inline source/ITimer.h:1]: We use lgpl license in rialto-gstreamer not apache

File: source/Timer.cpp [hunk 3]
```diff
@@ -1,315 +1,98 @@
 /*
- * Copyright (C) 2022 Sky UK
```
Comment (aczs) [inline source/Timer.cpp:2]: We use lgpl license in rialto-gstreamer not apache

File: source/Timer.h [hunk 4]
```diff
@@ -1,285 +1,72 @@
 /*
```
Comment (aczs) [inline source/Timer.h:1]: We use lgpl license in rialto-gstreamer not apache

File: source/Timer.h (hunk 1 above)
Comment (skywojciechowskim) [reply source/Timer.h]: removed

File: source/ITimer.h (hunk 2 above)
Comment (skywojciechowskim) [reply source/ITimer.h:1]: changed

File: source/Timer.cpp (hunk 3 above)
Comment (skywojciechowskim) [reply source/Timer.cpp:2]: changed

File: source/Timer.h (hunk 4 above)
Comment (skywojciechowskim) [reply source/Timer.h:1]: changed

**Review by skyshaha1 — APPROVED** [review]

**Review by aczs — APPROVED** [review]

**Pull request merged**